src/sql/parts/notebook/models/clientSession.ts

```diff
@@ -97,6 +99,11 @@ export class ClientSession implements IClientSession {
 			}
 			if (this._kernelPreference && this._kernelPreference.shouldStart) {
 				await this.startSessionInstance(this._kernelPreference.name);
+				return Promise.resolve();
```

Comment:
You don't need to do Promise.resolve() in either place, since for async methods it writes this for you.

Reply:
Thanks. Removed.